extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php

```diff
@@ -242,52 +246,98 @@ function () use ( $limit ) {
 		return $data;
 	}
 
+	private function getAllContributorsWithCurrentUser( $allContributorsData, $currentUserId ) {
+		global $wgCityId;
+
+		// Get current user's stats
+		$checkUserCallback = function( $user ) { return true; };
+		$userInfo = $this->wikiService->getUserInfo(
+			$currentUserId,
+			$wgCityId,
+			AvatarService::AVATAR_SIZE_SMALL_PLUS,
+			$checkUserCallback
+		);
+
+		if ($userInfo['lastRevision'] !== null) {
```

Comment:
Make sure all the code is spacey per conventions such as here and the foreach below.